data/config.json

```diff
@@ -1 +1 @@
-{"runningShowPath":"PastureBedtime.sparkle","runningScenePath":"Arttitude.scene"}
\ No newline at end of file
+{"runningShowPath":"PastureBedtime.sparkle","runningScenePath":"Arttitude.scene", "autoModeOn": true }
```

Comment:
Not sure this should be a persisted server setting.

If Pinky restarts, maybe we start in auto mode==`true` until we get some client interaction? Anyway, not a big deal either way, but it doesn't feel like important state to save between restarts.

Reply:
oic